design/resources/resources-mockup.json

```diff
@@ -30,6 +31,7 @@
             "averageRating": ""
          },
          {
+            "_id":"0216c7208a72e79b29997d0fd400beac",
```

Comment:
IDs are same in all json doc..I think it should be changed.. or not

Reply:
ids are not same they look similar but definitely they are different